src/engine/GroupBy.cpp

```diff
@@ -226,8 +226,7 @@ void processGroup(const GroupBy::Aggregate& a, size_t blockStart,
               } else {
                 // Remove the trailing character indicating if the value
                 // is an integer or a float.
-                res += ad_utility::convertIndexWordToFloat(
-                    entity.substr(0, entity.size() - 1));
+                res += ad_utility::convertIndexWordToFloat(entity);
```

Comment:
Given that we no longer remove the trailing character, we should probably update the comment.